spec/editors_draft.html

```diff
@@ -3298,6 +3298,244 @@ <h2>Evaluation Semantics</h2>
+      </p>
+
+      <ul>
+        <li>The parsing of a SPARQL Query string [[SPARQL11-QUERY]] or of a SPARQL Update string [[SPARQL11-UPDATE]], which may contain literals.</li>
+        <li>The loading of RDF data via a SPARQL <a data-cite="SPARQL11-UPDATE#load">`LOAD`</a> operation [[SPARQL11-UPDATE]]</li>
+        <li>The loading of RDF data via a single SPARQL graph store protocol request [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of RDF data via any single SPARQL <a data-cite="SPARQL11-QUERY#unnamedGraph">`FROM`</a> or <a data-cite="SPARQL11-QUERY#namedGraphs">`FROM NAMED`</a> clause [[SPARQL11-QUERY]] in systems that <a data-cite="SPARQL11-SERVICE-DESCRIPTION#sd-dereferencesuris">support the dereferencing and loading of RDF data specified for the dataset to be queried</a></li>
+        <li>Importing of any RDF data to the internally-managed RDF dataset via means other than SPARQL Update statements [[SPARQL11-UPDATE]] or the SPARQL Graph Store HTTP Protocol [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of a SPARQL result obtained from a SPARQL service when evaluating a <a data-cite="SPARQL11-FEDERATED-QUERY#service">`SERVICE`</a> pattern [[SPARQL11-FEDERATED-QUERY]]</li>
+      </ul>
+
+      <p>Loading the Turtle in <a href="#load-renaming-blank-example-simple"></a>, data with `_:b0` in a <a>cdt:List literal</a> MUST result in a system-internal representation of a single triple whose object is a <a>cdt:List literal</a> containing a blank node that, in the lexical form of the literal, is captured by a blank node identifier that is NOT "b0" (<a href="#load-renaming-blank-example-simple-result"></a>):
+      <pre id="load-renaming-blank-example-simple"
+         title="Loading of a cdt:List literal with a blank node"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      ex:s ex:p1 "[ _:b0 ]"^^cdt:List .
+      -->
+      </pre>
+      <pre id="load-renaming-blank-example-simple-result"
+         title="Loading of a cdt:List literal with a blank node: Result"
+         data-transform="updateExample"
+         data-content-type="text/n-triples"
+         class="nohighlight example">
+      <!--
+      <http://example.org/s> <http://example.org/p1> "[ _:b001 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      -->
+      </pre>
+
+      <p>Evaluating a SPARQL Update in <a href="#load-renaming-blank-example-load-twice"></a> with two `INSERT DATA` operations with the same blank-node-containing <a>cdt:List literal</a> MUST result in a system-internal representation of two triples whose <a>cdt:List literal</a> objects' <a>term lists</a> contain different blank nodes and where the literals' blank node identifiers MUST be different and MUST NOT be "b0" (<a href="#load-renaming-blank-example-load-twice-result"></a>):</p>
```

Comment:
Here is my edit suggestion from https://github.com/awslabs/SPARQL-CDTs/pull/2#discussion_r1584825928, directly applicable to the current version of this PR.
```suggestion
      <p>Evaluating a SPARQL Update in <a href="#load-renaming-blank-example-load-twice"></a> with two `INSERT DATA` operations with the same blank-node-containing <a>cdt:List literal</a> MUST result in a system-internal representation of two triples whose objects are <a>cdt:List literals</a> containing different blank nodes that, in the lexical form of these literals, are captured by different blank node identifiers (<a href="#load-renaming-blank-example-load-twice-result"></a>):</p>
```